actionpack/test/controller/integration_test.rb

```diff
@@ -170,9 +170,12 @@ def method_missing(name, *args)
 class IntegrationTestUsesCorrectClass < ActionDispatch::IntegrationTest
   def test_integration_methods_called
     reset!
+    headers = { "Origin" => "*" }
 
-    %w( get post head patch put delete ).each do |verb|
-      assert_nothing_raised { __send__(verb, "/") }
+    %w( get post head patch put delete options ).each do |verb|
+      public_send(RUBY_VERSION < "2.7" ? :assert_silent : :tap) do
```

Comment:
Can you remove this line? Ruby 2.6 doesn't raise any warnings for this delegation.

Reply:
Sorry didn't understand, yes I've removed this